app/Helpers/render/comment.php

```diff
@@ -92,8 +92,7 @@ function RenderCommentsComponent(
             ]);
     }
 
-    if (isset($user) && !$user->is_muted) {
-        // User comment input:
+    if ($user?->can('comment', [App\Models\Comment::class])) {
```

Comment:
I'm unable to comment on anything _except_ tickets. I think `comment` should be `create`, but that causes exceptions:
```
Too few arguments to function App\Policies\CommentPolicy::create(), 1 passed in /var/www/html/vendor/laravel/framework/src/Illuminate/Auth/Access/Gate.php on line 811 and exactly 2 expected
```

Reply:
Sorry about that, I incorrectly tested this code path. Fixed in latest.